odl/tomo/backends/astra_cuda.py

```diff
@@ -123,6 +122,11 @@ def astra_cuda_forward_projector_call(vol_data, geometry, proj_space,
         else:
             out.assign(tmp)
 
+    # Fix inconsistent scaling
+    if isinstance(geometry, Parallel2dGeometry):
+        # parallel2d scales with pixel stride
```

Comment:
It's only the 2D case where this rescaling is needed. So there are no anisotropic pixels.